src/applications/debt-letters/reducers/index.js

```diff
@@ -23,7 +24,7 @@ export const debtsReducer = (state = initialState, action) => {
         ...state,
         isPending: false,
         isError: false,
-        debts: action.debts,
+        debts: reverse(action.debts),
```

Comment:
These are coming in reverse order from the API based on their debt history dates